src/main/resources/com/example/style.css

```
@@ -0,0 +1,105 @@
+.root {
+    -fx-dark-red: #ff0038;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)